frontend/web/js/tests.js

```diff
@@ -52,21 +52,17 @@ function select_extension() {
 }
 
 
-// CUSTOM VARIANT SELECT
 function select_variant() {
-    const my_variant = [],
-        target_num = 2;
-
-    
-    let variant1 = document.getElementById('variant1');
-    let variant2 = document.getElementById('variant2');
-    if(variant1){
-        return variant1.value
-    }else{
-        return variant2.value
-    }
+    const target_num = 2;
 
-    
+    let variant = null;
+    for (let i =1; i<=target_num; ++i){
+        let temp_variant = document.getElementById(`variant${i}`);
+        if(temp_variant.checked){
+            variant = temp_variant.value
+        }
+    }
+    return variant
```

Comment:
Return the variable inside the `if` construct so that no unnecessary iterations take place